app/packages/core/src/context/PluginContext.tsx

```diff
@@ -113,18 +158,34 @@ export const PluginContextProvider: React.FunctionComponent<IPluginContextProvid
   };
 
   if (isLoading) {
-    return <div></div>;
+    return (
+      <Box minHeight="100vh" minWidth="100%" display="flex" flexDirection="column" justifyContent="center">
+        <Paper sx={{ display: 'inline-flex', mx: 'auto', p: 10 }}>
+          <CircularProgress />
+        </Paper>
+      </Box>
+    );
   }
 
   if (isError) {
-    return <div></div>;
+    return (
+      <Box minHeight="100vh" minWidth="100%" display="flex" flexDirection="column" justifyContent="center">
+        <Box sx={{ display: 'inline-flex', mx: 'auto', p: 10 }}>
+          <Alert severity="error" variant="outlined">
+            {error.message}
+          </Alert>
+        </Box>
+      </Box>
+    );
   }
 
   return (
     <PluginContext.Provider
       value={{
-        getInstance: getInstance,
-        getPlugin: getPlugin,
+        getAvailableClusters,
+        getAvailablePluginTypes,
+        getInstance,
+        getPlugin,
```

Comment:
Can we please always name the properties:

```suggestion
        getAvailableClusters : getAvailableClusters,
        getAvailablePluginTypes: getAvailablePluginTypes,
        getInstance: getInstance,
        getPlugin: getPlugin,
```